src/Types/Font.php

```diff
@@ -14,15 +14,15 @@ class Font
     public int $baseSize;
     public int $charsCount;
     public int $charsPadding;
-    public CData $texture;
+    public Texture2D $texture;
     public CData $recs;
     public CData $chars;
 
     public function __construct(
         int $baseSize,
         int $charsCount,
         int $charsPadding,
-        CData $texture,
+        Texture2D $texture,
```

Comment:
This broke `Raylib::loadFont()`. Please refactor it there and make sure that `examples/textures/textures_image_drawing.php` still works.